### PR TITLE
some README cleanup

### PR DESCRIPTION
**Pull request by Dan33l**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+This puppet module manage [keepalived](http://www.keepalived.org/). 
+The main goal of this keepalived is to provide simple and robust facilities for loadbalancing and high-availability to Linux system and Linux based infrastructures.
 
-## Tested on...
+## Setup
 
-* Debian 7 (Wheezy)
-* Debian 6 (Squeeze)
-* RHEL 6
+The module requires `puppetlabs/stdlib` and `puppetlabs/concat`, detailled in `metadata.json`.
```
Comment (alexjfisher) [inline README.md]: Spelling of ‘detailed’

Comment (alexjfisher) [reply README.md]: Probably not worth repeating hard dependencies anyway.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+2. [Setup - The basics of getting started with keepalived](#setup)
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Limitations - OS compatibility, etc.](#limitations)
+5. [Development - Guide for contributing to the module](#development)
 
-## Requirements
+## Description
 
-* [concat module](https://github.com/ripienaar/puppet-concat)
+This puppet module manage [keepalived](http://www.keepalived.org/).
```
Comment (alexjfisher) [inline README.md]: ‘manages’ (with an ‘s’)

File: README.md [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Limitations - OS compatibility, etc.](#limitations)
+5. [Development - Guide for contributing to the module](#development)
 
-## Requirements
+## Description
 
-* [concat module](https://github.com/ripienaar/puppet-concat)
+This puppet module manage [keepalived](http://www.keepalived.org/). 
+The main goal of this keepalived is to provide simple and robust facilities for loadbalancing and high-availability to Linux system and Linux based infrastructures.
```
Comment (alexjfisher) [inline README.md]: Remove the word ‘this’.

Comment (Dan33l) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (Dan33l) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -320,29 +318,22 @@ keepalived::lvs::real_server { 'example2.example.com':
 }
 ```
 
+## Reference
+
+TODO , Datatype and puppet-strings : This section is deprecated. Instead, add reference information to your code as Puppet Strings comments, and then use Strings to generate a REFERENCE.md in your module. For details on how to add code comments and generate documentation with Strings, see the Puppet Strings [documentation](https://puppet.com/docs/puppet/latest/puppet_strings.html) and [style guide](https://puppet.com/docs/puppet/latest/puppet_strings_style.html)
```
Comment (alexjfisher) [inline README.md]: Maybe just replace with
```
Reference documentation [coming soon](https://github.com/voxpupuli/puppet-keepalived/issues/158)
```
?? :)

File: README.md [hunk 5]
```diff
@@ -320,29 +318,22 @@ keepalived::lvs::real_server { 'example2.example.com':
...
-    $ bundle exec rake spec
+The contributing guide is in [CONTRIBUTING.md](https://github.com/voxpupuli/puppet-keepalived/blob/master/.github/CONTRIBUTING.md).
 
-Test against a specific Puppet or Facter version:
+## Release Notes/Contributors/Etc.
 
-    $ PUPPET_VERSION=3.2.1  bundle update && bundle exec rake spec
-    $ PUPPET_VERSION=2.7.19 bundle update && bundle exec rake spec
-    $ FACTER_VERSION=1.6.8  bundle update && bundle exec rake spec
+Details in `CHANGELOG.md`.
```
Comment (alexjfisher) [inline README.md]: Could you turn this into a link?

Comment (alexjfisher) [reply README.md]: oh... we don't actually have a changelog

**Pull request merged**